lib/imports/emails.js

```diff
@@ -32,21 +32,21 @@ ${signature}
 
 export const registrationInfoHTML = `
 <p>
-  <h4>Registration for the ${eventYear} ${siteName} is a 4-step process:</h4>
+  <h4>Registration is a 4-step process:</h4>
   <ol>
     <li>
       Create account at <a target="_blank" href="${siteURL}/register">${siteURL}/register</a>.<br/>
       Verify your email before continuing.<br />
       Deadline: <strong>11:59 PM ${registrationCloseDate}</strong>
     </li>
     <li>
-      Purchase ticket code(s) at <a target="_blank" href="https://commerce.cashnet.com/TheGreatPuzzleHunt${eventYear}">https://commerce.cashnet.com/TheGreatPuzzleHunt${eventYear}</a>.<br />
+      Acquire ticket code(s) at <a target="_blank" href="https://commerce.cashnet.com/TheGreatPuzzleHunt${eventYear}">https://commerce.cashnet.com/TheGreatPuzzleHunt${eventYear}</a>.<br />
       Ticket code(s) will be sent to your email.<br />
-      Deadline: <strong>9:30 AM ${eventDate}</strong>
+      Deadline: <strong>9:30 AM ${eventDay}, ${eventDate}</strong>
     </li>
     <li>
       Redeem ticket code(s) at <a target="_blank" href="${siteURL}/redeem">${siteURL}/redeem</a>.<br/>
-      Deadline: <strong>9:30 AM ${eventDate}</strong>
+      Deadline: <strong>9:30 AM ${eventDay}, ${eventDate}</strong>
```

Comment:
Do all the other instructions in here match what's in the FAQ?

Reply:
This is something I meant to ask about, the "Important Information" section of the email is a chunk that we used to have on the website but is no longer anywhere.

Reply:
Oh, weird. Which section is that?

Reply:
Lines 56-66:
`<h4>Important Registration Information:</h4>
  <ul>
    <li>
      Participants are welcome to purchase extra ticket codes for others to redeem.
      <ul>
        <li>Anyone who gains access to a ticket code may redeem it.</li>
        <li>Each ticket code may be used only once.</li>
      </ul>
    </li>
    <li>Each participant must redeem a ticket code to complete their registration.</li>
    <li>Each member of a team must complete their registration for the team to participate.</li>
  </ul>`

Reply:
Maybe we change it to "**acquire** extra ticket codes" and then leave as-is? I still like it even if this is the only place it shows up

Reply:
Sounds good! I'll make that change and merge.